src/main/java/com/example/demo/service/SQLMethods.java

```diff
@@ -0,0 +1,89 @@
+package com.example.demo.service;
+
+import com.example.demo.entity.Team;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import java.sql.*;
+
+@Service
+@AllArgsConstructor
+public class SQLMethods {
+    private static final String URL = "jdbc:mysql://localhost:3306/tournament_db";
+    private static final String USER = "root";
+    private static final String PASSWORD = "root";
+
+
+    public SQLMethods teamRegister(String teamName, String coach, String captain) {
+        Integer points = 0;
+        try (Connection connection = DriverManager.getConnection(URL, USER, PASSWORD)) {
+            PreparedStatement statement = connection.prepareStatement("INSERT into teams(team_name, coach, captain,points) values(?,?,?,?)");
```

Comment:
Good code style is that you use capitalized words when it is SQL syntax and lowercase words when it is something defined by you.
Example:
```
INSERT INTO teams (team_name) VALUES (?);
```